Web3Swift/TransactionBytes/EthManuallySignedTransactionBytes.swift

```diff
@@ -0,0 +1,90 @@
+//
+// This source file is part of the Web3Swift.io open source project
+// Copyright 2018 The Web3Swift Authors
+// Licensed under Apache License v2.0
+//
+// EthManuallySignedTransactionBytes.swift
+//
+// Created by Vadim Koleoshkin on 24/07/2018
+//
+
+import CryptoSwift
+import Foundation
+
+/** Signed transaction bytes */
+public final class EthManuallyTransactionBytes: BytesScalar {
```

Comment:
The naming doesn't correlate with what other `TransactionBytes` do. Currently all `TransactionBytes` are signed transactions that could be deployed to the network. Lets name it something different; these can be our key words - transaction rlp parameters.

Reply:
It actually correlates with the meaning of the class. 
`EthUnsignedTransactionBytes` — is used to create hex that is signed.
`EthManuallySignedBytes` — is used to create hex that could be passed to the network. The only difference with `EthTransactionBytes` is that we pass signature (rsv) outside of the class.